### ISSUE TITLE
`theme` directive emits error when using alpha level for color

### ISSUE THREAD

**Issue by zkulbeda** [tailwindlabs/tailwindcss-intellisense#614]
**What version of Tailwind CSS IntelliSense are you using?**

v0.8.7

**What version of Tailwind CSS are you using?**

v3.1.8

**What operating system are you using?**

Windows

**Reproduction**

```css
div{
  background-color: theme("colors.blue.900 / 90%");
}
```

**Describe your issue**

As mentioned in [docs](https://tailwindcss.com/docs/functions-and-directives#functions), I can use slash to specify alpha level for color. But vscode consider it as wrong syntax. Project compiles without errors as was expected.

![image](https://user-images.githubusercontent.com/24959171/191822606-b2b5605d-6d7a-4390-8d07-5256afdf44f5.png)

![image](https://user-images.githubusercontent.com/24959171/191822650-ea0b953b-2a6c-4a55-a768-af538bb29323.png)

**Comment by davidjbng** [comment]
As a workaround I found this syntax is supported by both Tailwind and IntelliSense
```
div {
   background-color: theme("colors.blue.900" / 90%);
}
```

**Comment by bradlc** [comment]
Hey @zkulbeda, thanks for reporting this. This issue is fixed in `v0.9.0`, via 1b730cb6568d8e19b160fe25bdb0b6f8157d174c 👍
